README.md

```diff
@@ -123,7 +126,6 @@ No deployment instructions yet.
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228913458-d13e63c3-d1dd-451d-bee4-59281bbe8c70.png)

- [ ] Kindly remove the Live Demo section because you don't have any live link for your application 🤔 Once you would have a live link then you can add this link to make your readme have the correct and up to date information about your project 👍